docs/how-to/hip_runtime_api.rst

```diff
@@ -0,0 +1,35 @@
+.. meta::
+  :description: HIP runtime API usage
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime API How to,
+
+.. _hip_runtime_api_how-to:
+
+********************************************************************************
+HIP runtime API
+********************************************************************************
+
+The HIP runtime API provides C and C++ functionalities to manage event, stream,
+and memory on GPUs. On the AMD platform, the HIP runtime uses
+:doc:`Common Language Runtime (CLR) <hip:understand/amd_clr>`, while on NVIDIA
+CUDA platform, it is only a thin layer over the CUDA runtime or Driver API.
+
+- **CLR** contains source code for AMD's compute language runtimes: ``HIP`` and
+  ``OpenCL™``. CLR includes the ``HIP`` implementation on the AMD
+  platform: `hipamd <https://github.com/ROCm/clr/tree/develop/hipamd>`_ and the
+  Radeon Open Compute Common Language Runtime (``rocclr``). ``rocclr`` is a
```

Comment:
ROCm Compute Language Runtime (``rocclr``)

Reply:
I pushed your suggestion, however, the ROCm Common Language Runtime in this case would make sense also, because rocclr contains the compute runtime used in HIP and OpenCL, so it's common in HIP and OpenCL.

@jujiang-del please double-check at your side, that we are sure that "Compute" is the correct.

Reply:
Yes, please use "Compute"